frontend/src/layout/navigation-3000/sidepanel/panels/SidePanelSdkDoctor.tsx

```diff
@@ -128,7 +128,7 @@ const COLUMNS: LemonTableColumns<AugmentedTeamSdkVersionsInfoRelease> = [
     {
         title: (
             <span>
-                LAST EVENT AT{' '}
+                EVENT TIME{' '}
```

Comment:
Hm, this one doesn't look very obvious to me. How about just "LAST EVENT"? Same amount of characters saved